pkg/customcost/pipelineservice_test.go

```diff
@@ -0,0 +1,195 @@
+package customcost
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"runtime"
+	"testing"
+	"time"
+
+	"github.com/opencost/opencost/core/pkg/log"
+	"github.com/opencost/opencost/core/pkg/util/timeutil"
+	"github.com/opencost/opencost/core/pkg/version"
+)
+
+func TestPipelineService(t *testing.T) {
+	// establish temporary test assets dir
+	dir, err := os.MkdirTemp("", "ocplugin-*")
```

Comment:
Could we update this to use `t.TempDir()`?

Reply:
good thinking